tools/icu/icu-generic.gyp

```diff
@@ -30,12 +34,8 @@
       'toolsets': [ 'host', 'target' ],
       'direct_dependent_settings': {
         'defines': [
-          'UCONFIG_NO_LEGACY_CONVERSION=1',
-          'UCONFIG_NO_IDNA=1',
-          'UCONFIG_NO_TRANSLITERATION=1',
-          'UCONFIG_NO_SERVICE=1',
-          'UCONFIG_NO_REGULAR_EXPRESSIONS=1',
           'U_ENABLE_DYLOAD=0',
+          'UCONFIG_NO_SERVICE=1',
```

Comment:
I don't remember exactly what is the goal of these changes. Is it to save space in the target executable and in memory?